### PR TITLE
[CM-987] - Migrating to Capacitor 3

### PR DESCRIPTION
**Pull request by amntoppo**
Migrated both Android and iOS

### CODE REVIEW

File: example/android/app/src/main/java/io/ionic/starter/MainActivity.java [hunk 1]
```diff
@@ -1,27 +1,20 @@
...
+    super.onCreate(savedInstanceState);
 
-        // Initializes the Bridge
-        Log.w("KmService", "Token saved : " + FirebaseInstanceId.getInstance().getToken());
-        this.init(savedInstanceState, new ArrayList<Class<? extends Plugin>>() {{
-            add(KommunicateCapacitorPlugin.class);
-            // Additional plugins you've installed go here
-            // Ex: add(TotallyAwesomePlugin.class);
-        }});
-    }
```
Comment (Sathyan-Elangovan) [inline example/android/app/src/main/java/io/ionic/starter/MainActivity.java:26]: Why we have removed `FirebaseInstanceId.getInstance().getToken())`

File: example/src/app/home/home.page.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import { Component } from '@angular/core';
-import 'capacitor-plugin-kommunicate';
+import {KommunicateCapacitorPlugin} from 'capacitor-plugin-kommunicate';
 import { Plugins } from '@capacitor/core';
 import { Capacitor } from '@capacitor/core';
-const { KommunicateCapacitorPlugin, Modals } = Plugins;
+const {Modals } = Plugins;
 import { LoadingController } from '@ionic/angular';
-const appId = "<Your-App-Id>"
+const appId = "eb775c44211eb7719203f5664b27b59f"
```
Comment (Sathyan-Elangovan) [inline example/src/app/home/home.page.ts:7]: Don't commit the app id. Remove this

**Review by Sathyan-Elangovan — CHANGES_REQUESTED** [review]

File: example/android/app/src/main/java/io/ionic/starter/MainActivity.java (hunk 1 above)
Comment (amntoppo) [reply example/android/app/src/main/java/io/ionic/starter/MainActivity.java:26]: It was a log, which might fail and throw error in the sample app.

File: example/src/app/home/home.page.ts (hunk 2 above)
Comment (amntoppo) [reply example/src/app/home/home.page.ts]: Updated

**Review by Sathyan-Elangovan — APPROVED** [review]

**Pull request merged**